tests/scripts/task_build_hexagon_api.sh

```diff
@@ -19,8 +19,18 @@
 set -e
 set -u
 
+use_cache=false
+if [ $# -ge 1 ] && [[ "$1" == "--use-cache" ]]; then
+    use_cache=true
+    shift 1
+fi
+
```

Comment:
Unrelated?

Reply:
Unrelated to this PR but useful change for development. I could send a separate PR if you suggest.